app/models/user.rb

```diff
@@ -34,7 +37,7 @@ def send_devise_notification(notification, *args)
 
   def queue_reminder_email
     return if reminder_sent_at
-    UserMailer.incomplete_reminder_email(id).deliver_later(wait: 1.day)
```

Comment:
Whoops! Need to remove this.

Reply:
Remove the whole line? That’s in place to remind users to finish applying if they haven’t within 24 hours of sign-up. What we could do is cancel the job if they’re promoted to a different role.

Reply:
Whoops, sorry; I meant I needed to remove the change that commented that line out. 
Although that is an interesting idea... I'll add that to a permissions issue.